fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -324,12 +322,10 @@ protected void touchParent( final String id ) {
 
     /* Overriding so unit test can mock. */
     @Override
-    @VisibleForTesting
```

Comment:
I'm not sure I even want to remove the annotation. What is the advantage of so doing and adding Javadocs? Is it the Guava dependency that bugs you, @acoburn ?